src/Otp.php

```diff
@@ -307,7 +307,7 @@ private function getBinaryCounter($counter)
      */
     private function getTimecounter()
     {
-        return floor((time() + $this->totpOffset) / $this->period);
+        return \intval(floor((time() + $this->totpOffset) / $this->period));
```

Comment:
`floor` gives a float tpye that actually has an integer value. `intval`()` turns into into an actual `int` type, which is what the return type is supposed to be.